### PR TITLE
[collapsible-list] add FAQ toggle

### PR DESCRIPTION
**Pull request by tamrat-bay**

### CODE REVIEW

File: examples/main/shared/editor/EditorPlugins.tsx [hunk 1]
```diff
@@ -594,6 +594,7 @@ export const config: RichContentEditorProps['config'] = {
   },
   [COLLAPSIBLE_LIST_TYPE]: {
     innerRCEPlugins,
+    isQapageData: true,
```
Comment (tamrat-bay) [inline examples/main/shared/editor/EditorPlugins.tsx:597]: add docu

Comment (danarish4) [inline examples/main/shared/editor/EditorPlugins.tsx:597]:
```suggestion
    isQaPageData: true,
```

File: packages/plugin-collapsible-list/web/src/components/modal-components/CollapsibleListSettings.jsx [hunk 2]
```diff
@@ -127,6 +130,64 @@ class CollapsibleListSettings extends Component {
...
+            <InfoIcon onMouseEnter={showTooltip} onMouseLeave={hideTooltip} />
+            <Popover active={active} parent={parent}>
+              {t('CollapsibleList_CollapsibleListSettings_Tab_Settings_FAQ_Enabled_Tooltip')}{' '}
+              {this.renderLearnMoreLink()}
+            </Popover>
+          </div>
+        )}
+      </PopoverWrapper>
+    );
+  };
```
Comment (danarish4) [inline packages/plugin-collapsible-list/web/src/components/modal-components/CollapsibleListSettings.jsx:164]: move to `MoreLink` file

File: packages/plugin-collapsible-list/web/src/components/modal-components/CollapsibleListSettings.jsx [hunk 3]
```diff
@@ -127,6 +130,64 @@ class CollapsibleListSettings extends Component {
...
+
+  renderQapageToggle = () => {
+    const { getDataManager, t, isMobile, theme } = this.props;
+    return (
+      <>
+        {this.renderSeparator()}
+        <LabeledToggle
```
Comment (danarish4) [inline packages/plugin-collapsible-list/web/src/components/modal-components/CollapsibleListSettings.jsx:171]: maybe you can create a `Toggle` UI component that can be reused both inside LabeledToggle and in this case.

**Review by danarish4 — CHANGES_REQUESTED** [review]
You can edit `migration-content.json` to include collapsible with the new attribute (this will cover tests - we can also do it for link seo), you can also open an experiment for the testApp so we can earn E2E.

File: packages/toolbars-modals/web/src/link/LinkComponents/AdvancedCheckboxes.tsx [hunk 4]
```diff
@@ -56,14 +56,14 @@ const AdvancedCheckboxes: React.FC<Props> = ({
               />
               {isMobile && useAdvancedSettings ? (
                 <div className={classNames(styles.linkPanel_advancedSettings_mobile_text_info)}>
-                  {t(checkbox.tooltipTextKey)} {renderLearnMoreLink()}
+                  {t(checkbox.tooltipTextKey)} {renderMoreLink()}
```
Comment (danarish4) [inline packages/toolbars-modals/web/src/link/LinkComponents/AdvancedCheckboxes.tsx:59]:
```suggestion
                  {t(checkbox.tooltipTextKey)}
                  <MoreLink link={learMoreUrl} t={t} />
```

this split is not necessary, I this it is more readable that way

**Review by danarish4 — APPROVED** [review]
Please dont forget to change migration-content to include collapsible with `isQapageData` and link with `noreferrer`

**Pull request merged**